src/App.tsx

```diff
@@ -1,13 +1,16 @@
 import GlobalStyle from "./assets/globalStyles/GlobalStyle";
 import Header from "./cmps/Header";
 import PokemonContent from "./cmps/PokemonContent";
+import { useState } from "react";
 
 function App() {
+  const [selectedCtg, setSelectedCtg] = useState(0);
+
```

Comment:
the app should be clean, create a new component and the state management will be there